sig-ui/annual-report-2021.md

```diff
@@ -0,0 +1,102 @@
+# SIG UI 2021 Annual Report
+
+**Authors:**
+
+Sebastian Florek ([@floreks](https://github.com/floreks))
+Marcin Maciaszczyk ([@maciaszczykm](https://github.com/maciaszczykm))
+Jeffrey Sica ([@jeefy](https://github.com/jeefy))
+
+- [Operational](#operational)
+- [Membership](#membership)
+- [Current initiatives and project health](#current-initiatives-and-project-health)
+
+## Operational
+
+**How are you doing with operational tasks in [sig-governance.md](/committee-steering/governance/sig-governance.md)?**
+
+The SIG primarily works asynchronously. In the past, our meetings would last less than five minutes and be relatively disruptive. Since the release of the 2.0 Dashboard, meeting attendance transitioned from optional to near-non-existent. Instead, we coordinate through the #sig-ui Slack channel and handle any project planning via GitHub. All chairs are regular contributors of the Dashboard subproject and help review PRs and triage issues. 
+
+**Is your README accurate? have a CONTRIBUTING.md file?**
+
+Yes, our README is accurate and is typically updated with every new Dashboard release. Also we do have a CONTRIBUTING.md file however it has not been updated with any more info beyond the standard template.
+
+**All subprojects correctly mapped and listed in [sigs.yaml](/sig-list.md)?**
+
+Yes. The only subproject under SIG-UI is the Kubernetes Dashboard. There is an additional companion service that gets deployed to feed metrics into the Dashboard (Dashboard Metrics Scraper) but it isn't considered a subproject by the SIG.
+
+**What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?**
+
+As stated above, the SIG currently doesn't hold any meetings and instead has been asynchronous for roughly a year. All communication is done via the #sig-ui Slack channel and all planning activities are managed within the Dashboard GitHub repo.
+
+**How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?**
+
+As we have no subproject other than the Dashboard, there is no real feedback loop besides GitHub Issues and Slack channel traction.
+
+**When was your last monthly community-wide update? (provide link to deck and/or recording)**
+
+Slide Deck: https://docs.google.com/presentation/d/1W4NioOkAF2VFiu-5t80p2vlu3_OznpugiyiViFuitaM/edit?usp=sharing
```

Comment:
I'd call out that this occurred in May 2020. You can't see it unless you open the links.

Reply:
@jeefy Can you put this into the doc? Unfortunately with @floreks we are not able to edit the file.